symcrypt-bindgen/build.rs

```diff
@@ -41,6 +41,15 @@ fn main() {
         .allowlist_function("^(SymCryptEckey(Allocate|Free|SizeofPublicKey|GetValue|SetRandom|SetValue|SetRandom|))$")
         .allowlist_var("SYMCRYPT_FLAG_ECKEY_ECDH")
         .allowlist_function("SymCryptEcDhSecretAgreement")
+        // RSA FUNCTIONS
+        .allowlist_function("^(SymCryptRsakey.*)$")
+        .allowlist_function("^(SymCryptRsaRaw.*)$")
+        // DSA FUNCTIONS
+        .allowlist_function("^(SymCryptDsa(Sign|Verify).*)$")

```

Comment:
Why Dsa instead of EcDsa, EcDh

Reply:
EcDh is already added from the previous work , Will add in EcDsa as well.